src/test/java/de/hhu/propra/link/controllers/LinkControllerTest.java

```diff
@@ -26,9 +26,15 @@ void testIndex() throws Exception {
 
     @Test
     void testNewLink() throws Exception {
-        mvc.perform(post("/")
+        mvc
+                .perform(
+                        post("/")
+                                .param("abbreviation", "abc")
+                                .param("url", "http://www.abc.de")
+                )
+                .andExpect(status().is3xxRedirection());
                 .param("abbreviation", "abc")
```

Comment:
This needs to go - I blame GitHub

Reply:
Sorry, I didn't check your changes probably.